src/shared/locales/en/common.json

```diff
@@ -59,6 +60,11 @@
     "blockchainIsNotClaimable": "The account blockchain is not claimable: {{address}} {{blockchain}}",
     "loginSessionIsNotDefined": "Unknown error. Login session is not defined."
   },
+  "dateFormats": {
+    "dateHourMinutes": "MM/dd/yyyy HH:mm",
+    "hourMinutes": "HH:mm",
+    "extendedDate": "dd MMM yyyy"
+  },
```

Comment:
Wouldn't it be better if these formats were inside the pages or components lang that use them? So when we need to change it doesn't affect other parts